### PR TITLE
modules: add new missing screenshots

### PR DESCRIPTION
**Pull request by lasers**
Nine-plus (9+) screenshots.
* 🖼 🖼 🖼
* 🖼 🖼 🖼
* 🖼 🖼 🖼 ➕

### CODE REVIEW

File: py3status/modules/weather_owm.py [hunk 1]
```diff
@@ -233,24 +233,28 @@
...
 @licence MIT
+
+SAMPLE OUTPUT
+{'full_text': 'New York 🌫 ○ 30°F, mist ☁ ☁ ☁'}
+
+new_weather
+{'full_text': '🌫 ○ 59°F, foggy ⛅ ☼ 🌧'}
+
+# Currently foggy, 59°F outside, with forecast of cloudy
+# tomorrow, sunny the next day, then rainy in two days.
```
Comment (tobes) [inline py3status/modules/weather_owm.py]: I don't trust these comments here.

File: py3status/modules/weather_owm.py [hunk 2]
```diff
@@ -233,24 +233,28 @@
...
 
-Example configuration:
+Examples:
 ```
+# change icons
 weather_owm {
-  api_key = '...'
-
-  icons {
+  icons = {
```
Comment (tobes) [inline py3status/modules/weather_owm.py]: This is should not be part of this PR as it is not related to screenshots.

I will accept it but you need to make your PRs cleaner.  If you do this then they will get actioned far faster.  Good catch though.

File: py3status/modules/graphite.py [hunk 3]
```diff
@@ -66,6 +66,9 @@
     color_degraded: threshold_degraded has been exceeded
 
 @author ultrabug
+
+SAMPLE OUTPUT
+{'color': 'full_text': 'graphite'}
```
Comment (tobes) [inline py3status/modules/graphite.py]: I'm not convinced by this output.  I may try to get some real data

**Comment by lasers** [comment]
>Can you confirm if you have built the documentation using this branch.

Can't. I'm missing `fonttools` on `Solus`.

**Comment by tobes** [comment]
> >  Can you confirm if you have built the documentation using this branch.
>
> Can't. I'm missing fonttools on Solus.

Ok well I'll see if I can do a test build at some point.  I'm pretty sure your comments will break stuff.

Is this all faked output?  Or is some of it real - as in actually produced by the module

**Comment by lasers** [comment]
I faked everything. I looked at this again.

In `systemd`, the real is probably `sshd.service: active`. Fixed.

`graphite` is not `graphite`. :-\

`icinga2` is missing at least `4: INVALID STATUS`. Not sure if we want to add error code too.... Also, this could be superseded by `getjson`.

I think I'm right on target with `glpi`, `ns_checker`.

In `pingdom`, I'm sure it is `{server_name}: 323ms` and if more than one, then `{server_name}: 322ms, {server_name}: 341ms`. I use `Pingdom` as server name. Maybe we should use `py3status` ? Nah. 

I vaguely recall `thunderbird_calendar` is what it is, but we should merge `thunderbird_todos` anytime before the release to avoid any depreciation. So no worries.

I removed the comments because you don't trust the weather predictions.

File: py3status/modules/graphite.py (hunk 3 above)
Comment (ultrabug) [reply py3status/modules/graphite.py]: yeah switch to '412 req/s' instead as graphite is meant to return numbers with units usually

**Comment by ultrabug** [comment]
just change graphite and let's go

**Comment by lasers** [comment]
`graphite` updated.

**Comment by tobes** [comment]
@lasers this PR has broken screenshots (I'll fix them) can you please try to test your PRs before you make them?  You need to take much more care with your code.

**Comment by lasers** [comment]
>Can you confirm if you have built the documentation using this branch.
>>Can't. I'm missing fonttools on Solus.

I will try to switch a distro so I can test the docs. I need to be more careful with amend tweaks (wifi).

**Comment by tobes** [comment]
> I need to be more careful with amend tweaks

You do.  As long as you learn from these things then all is good.  This is also why you should make smaller more focused PRs as they are easier to review and catch these issues

**Pull request merged**